jingo_minify/management/commands/compress_assets.py

```diff
@@ -5,13 +5,11 @@
 import shutil
 import time
 import urllib2
-from subprocess import call, PIPE
+from subprocess import call, check_output, PIPE
```

Comment:
I'm pretty sure this isn't an issue, but this does mean it's no longer Python 2.6 compatible.